### ISSUE TITLE
Make this.autorun trigger angular digest

### ISSUE THREAD

**Issue by hlehmann** [Urigo/angular-meteor#1029]
When operations occur in a autorun, the result isn't digest by angular.

Currently I use it that way but maybe this.autorun should always call $scope.$apply()
```
this.autorun(() => {
  ...
  $scope.$$phase || $scope.$apply();
}
```

**Comment by Urigo** [comment]
https://github.com/Urigo/angular-meteor/pull/1058 should fix that, can you test with current master?

**Comment by Deadly0** [comment]
@Urigo how to test it with current master? Looks like i have the same issue.

**Comment by sklink** [comment]
I have `1.3.4` and am experiencing the same issue.

**Comment by Urigo** [comment]
Fixed in `1.3.5`
